vspace/src/main/java/edu/asu/diging/vspace/web/staff/SlideController.java

```diff
@@ -27,12 +29,13 @@
 
     @RequestMapping("/staff/module/{moduleId}/slide/{id}")
     public String listSlides(@PathVariable("id") String id, @PathVariable("moduleId") String moduleId, Model model) {
-
+        
         model.addAttribute("module", moduleManager.getModule(moduleId));
         model.addAttribute("slide", slideManager.getSlide(id));
         model.addAttribute("slideContents", contentBlockManager.getAllContentBlocks(id));
-
+        if(slideManager.getSlide(id) instanceof BranchingPoint) 
+            model.addAttribute("choices", ((IBranchingPoint)slideManager.getSlide(id)).getChoices());
```

Comment:
please follow our style guide

Reply:
put `slideManager.getSlide(id)` into a varialbe. there is no reason to call the db twice (even though it might be cached)